baseband/mark5b/base.py

```diff
@@ -451,4 +455,13 @@ def _make_frame(self, index):
     :class:`~baseband.mark5b.base.Mark5BFileWriter` (binary), or
     :class:`~baseband.mark5b.base.Mark5BStreamReader` or
     :class:`~baseband.mark5b.base.Mark5BStreamWriter` (stream).
+
+Notes
+-----
+One can also pass in a list, tuple, or subclass of
```

Comment:
For Mark 5B, I can more easily believe it will be used in the wild, so no need to add a comment here.